app/views/events/show.html.erb

```diff
@@ -36,3 +36,8 @@
 
 <%= link_to 'Edit', edit_event_path(@event) %> |
 <%= link_to 'Back', events_path %>
+<% if @event.creator_id == @current_user.id %>
+  <td><%= button_to 'Destroy', @event, method: :delete,data: { confirm: 'Are you sure?' }, class: 'btn btn-danger'%></td>
```

Comment:
Same as above comment

Reply:
I finished the modification and I made a new push